io.openems.backend.alerting/src/io/openems/backend/alerting/scheduler/MessageSchedulerService.java

```diff
@@ -3,6 +3,13 @@
 import io.openems.backend.alerting.Handler;
 import io.openems.backend.alerting.Message;
 
+/**
+ * Specifies classes able to generate a fitting {@link MessageScheduler} to a
+ * {@link Handler} with the same generic {@link T}.
+ *
+ * @author kai.jeschek
```

Comment:
We do not use the `@author` code in OpenEMS, as this information is available via Git log.